### PR TITLE
enable reversed locale layout in QML

### PR DESCRIPTION
**Pull request by mgallien**
Signed-off-by: Matthieu Gallien <matthieu.gallien@nextcloud.com>

### CODE REVIEW

**Comment by mgallien** [comment]
reversed main dialog
image

**Comment by mgallien** [comment]
main dialog
image

File: src/gui/UserStatusSelectorDialog.qml [hunk 1]
```diff
@@ -20,6 +22,9 @@ Window {
 
     flags: Qt.Dialog
     
+    LayoutMirroring.enabled: Qt.application.layoutDirection === Qt.RightToLef
```
Comment (claucambra) [inline src/gui/UserStatusSelectorDialog.qml:25]: Is this a misspelling? :)

**Comment by claucambra** [comment]
The username label in the user button that invokes the dropdown also needs its alignment reversed

**Comment by jancborchardt** [comment]
> reversed main dialog
> image
> 

All the text needs to be right-aligned for RTL. Otherwise looks good! :)

**Comment by rodrigoscoelho** [comment]
> AppImage file: [Nextcloud-PR-4240-315cb6aa32283b0b6384f688b942baf537da4427-x86_64.AppImage](https://github.com/nextcloud-desktop-bot/ci-builds/releases/download/PR-4240/Nextcloud-PR-4240-315cb6aa32283b0b6384f688b942baf537da4427-x86_64.AppImage) To test this change/fix you can simply download above AppImage file and test it. Please make sure to quit your existing Nextcloud app and backup your data.

May you explain how to test this image?

Is it a .exe? Just rename? 

thanks

**Comment by claucambra** [comment]
> > AppImage file: [Nextcloud-PR-4240-315cb6aa32283b0b6384f688b942baf537da4427-x86_64.AppImage](https://github.com/nextcloud-desktop-bot/ci-builds/releases/download/PR-4240/Nextcloud-PR-4240-315cb6aa32283b0b6384f688b942baf537da4427-x86_64.AppImage) To test this change/fix you can simply download above AppImage file and test it. Please make sure to quit your existing Nextcloud app and backup your data.
> 
> May you explain how to test this image?
> 
> Is it a .exe? Just rename?
> 
> thanks

AppImage executables are Linux-only

**Comment by claucambra** [comment]
This needs a rebase :)

**Review by camilasan — APPROVED** [review]

**Comment by claucambra** [comment]
These labels need their alignment changed in the Right-To-Left layout to match how they are laid out in the Left-To-Right layout:

Screenshot 2022-05-04 at 11 46 34

Screenshot 2022-05-04 at 11 53 46

Once rebased the layout of the activity item contents are also not right-aligned when in Right-To-Left:

Screenshot 2022-05-04 at 11 55 52

**Comment by claucambra** [comment]
The unified search also doesn't seem to respond to RTL layout:

Screenshot 2022-05-04 at 12 04 39

**Comment by mgallien** [comment]
image
after rebase

**Pull request merged**